docs/Repository.md

```diff
@@ -309,6 +317,12 @@ A basic ADSL only BCM6362 based gateway. Very useful as SIP ATA.
 
 ## DGA4130 / VBNT-K
 
+### MST
+
+| Type   | Version   | Timestamp  | Root Strategy | Mirror |
+|:------:|:----------|:-----------|:--------------|:-------|
+| ???    | 17.3.0165 | 2018-02-28 | -             | [HTTPS](https://github.com/hack-technicolor/hack-technicolor/raw/master/firmware/MST%20DGA4130%2017.3.0165.1681007.rbi?raw=true) |
```

Comment:
Same blob vs raw dilemma

Reply:
Isn't this Type 2 for strategy #D?

Reply:
idk for strategy

Reply:
#D is the same as for UNO's vant-6, did anybody test it on this firmware?